Manual.md

```diff
@@ -743,6 +743,8 @@ used.
 
 - `fetch_cmd` Executable to be used to fetch URLs in `distfiles` during the `do_fetch` phase.
 
+- `changelog` A string pointing to the `upstream` changelog.
```

Comment:
Not sure "upstream" should be formatted as code. I think this should also document that raw text files are preferred.

Reply:
Let's also be clear that the string is a URL (not a filename to changelog in the distfiles).

Reply:
I just copied description for homepage and changed it to changelog. I'll also change homepage to make it consistent.